src/components/DataDictionaryComponent/DataDictionary/table/DataDictionaryPropertyTable/component/ListComponent.js

```diff
@@ -37,7 +37,9 @@ const twoColumnsView = {
       },
       gutters: {
         paddingLeft: '2px',
-        marginBottom: '-4px',
+        // marginBottom: '-4px',
```

Comment:
@rana22 Can you remove this comment if not required.

Reply:
updated